backend/tracim_backend/lib/core/event.py

```diff
@@ -93,6 +100,21 @@ def _base_query(
         else:
             # ALL doesn't need any filtering an is the only other handled case
             assert read_status == ReadStatus.ALL
+
+        if event_types:
+            event_type_filters = []
+            for event_type in event_types:
+                event_type_filter = and_(
+                    Event.entity_type == event_type.entity,
+                    Event.operation == event_type.operation,
+                    Event.entity_subtype == event_type.subtype,
+                )
+                event_type_filters.append(event_type_filter)
+
+            if len(event_type_filters) > 1:
+                query = query.filter(or_(*event_type_filters))
+            else:
+                query = query.filter(event_type_filters[0])
```

Comment:
Would this work, to avoid a special case? Or does it add an overhead to call `or_` "uselessly"? 
```suggestion
            query = query.filter(or_(*event_type_filters))
```

I'm okay either way, though.

Reply:
Unfortunately this doesn't work.